### PR TITLE
docs: hpmn setup

### PR DESCRIPTION
**Pull request by thephez**
Preview: https://dash-docs--195.org.readthedocs.build/en/195/docs/user/masternodes/setup-hpmn.html

### CODE REVIEW

**Comment by thephez** [comment]
@strophy You think it's okay to merging this as-is (Core instructions only) to be updated once dashmate, etc. are functional? It's based on the testnet/mainnet setup pages. I've "commented out" all the DMT stuff because I don't know if it supports HPMNs at this point.

File: docs/user/masternodes/setup-hpmn.rst [hunk 1]
```diff
@@ -0,0 +1,716 @@
...
+
+- Manual installation (this guide)
+- `xkcd's installation guide <https://www.dash.org/forum/threads/system-wide-masternode-setup-with-systemd-auto-re-start-rfc.39460/>`__
+
+Manual installation
+-----------------------------
+
+To manually download and install the components of your Dash masternode,
+visit the `GitHub releases page <https://github.com/dashpay/dash/releases>`_ 
+and copy the link to the latest ``x86_64-linux-gnu`` version. Go back to
```
Comment (strophy) [inline docs/user/masternodes/setup-hpmn.rst:224]:
```suggestion
and copy the link to the latest version appropriate for your CPU architecture, e.g. ``x86_64-linux-gnu``. Go back to
```

**Comment by strophy** [comment]
I think this is fine, the instructions are mostly the same and this should be enough for most MNOs to get through the process based on what they already understand. We can add a dashmate section once dashmate development is ready.

**Comment by thephez** [comment]
> I think this is fine, the instructions are mostly the same and this should be enough for most MNOs to get through the process based on what they already understand. We can add a dashmate section once dashmate development is ready.

Sounds good. I've incorporated your suggestion :+1:

**Review by strophy — APPROVED** [review]

**Pull request merged**